File: 1598-crawler-log-folder/1598-crawler-log-folder.c
int minOperations(char** logs, int logsize){
    int beg = 0;
    int cur = 0;
    for (int i = 0; i < logsize;  i++)
    {
        printf("%s\n", logs[i]);
        if ((strcmp(logs[i], "../") == 0 && cur == 0) || strcmp(logs[i], "./") == 0){
            continue;
        }
        if (strcmp(logs[i], "../") == 0 && cur != 0){
            cur--;
        }
        else{
            cur++;
        }
    }
    return cur - beg;
}

File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        begptr = 0
        curptr = 0
        for i in range(len(logs)):
            if (logs[i] == "../" and curptr == 0) or logs[i] == "./":
                continue
            elif logs[i] == "../" and curptr != 0:
                curptr -= 1
            else:
                curptr += 1
        return curptr - begptr
